common/on_actions/hfe_on_actions.txt

```diff
@@ -124,18 +125,38 @@ on_entering_system_first_time = {
 		hfe.410
 		# copernicus
 		hfe.355
+		# drones survey
+		hfe_maintenance.3
```

Comment:
Why it here

Reply:
It shouldn't exist. When the Drones awaken, they instantly survey things, thereby removing all Anomalies and other things for the rest of the galaxy. It's the exact same problem that the Pirates had.